_features/probe_temp_compensation.md

```diff
@@ -50,20 +50,17 @@ While probe calibration is active bed temperature is held constant (_e.g.,_ 110
    - Wait until the probe reaches the new target temperature.
    - Probe the bed to get a delta value.
  - In the case of a timeout, compensation values for higher temperatures will be extrapolated from the existing values.
- 
+
 ## Configuration
 1. Make sure you have a heated bed and a probe with thermistor.
 2. Enable option for probe + bed compensation + calibration:
     - `PROBE_TEMP_COMPENSATION`
-3. Set the maximum temperature that can be reached by your heated bed:
-    - `PTC_MAX_BED_TEMP`
+3. Ensure the maximum temperature that can be reached by your heated bed is set in `Configuration.h`:
+    - `BED_MAXTEMP`
 3. Set the park position to wait for probe to cool down:
```

Comment:
It's not the maximum temperature the bed can reach (max put in energy - losses at that temperature = zero). It's the temperature the bed can stand without destroying itself.
My 220V AC 500W beds are able to destroy themselves in seconds.

Reply:
I only slightly adjusted the language from what was there before. What would you suggest instead?

Reply:
I've simplified the `BED_MAXTEMP` instruction in https://github.com/MarlinFirmware/MarlinDocumentation/pull/315/commits/f44c4e183e51fe0573de78a74a8e1f7e0fc2186b.